SECURITY.md

```diff
@@ -201,6 +201,13 @@ the community they pose.
   that artifact is large enough to impact performance or
   cause the runtime to run out of resources.
 
+#### Vulnerabilities affecting software downloaded by Corepack
+
+* Corepack defaults to downloading the latest version of the software requested
```

Comment:
Should we call out that it doesn't always download from npm?

This is unique from downloading package managers with npm

Reply:
I'm not sure there is the assumption that downloads are from npm, so I'm good either way.